portfolio/src/main/java/com/google/sps/servlets/CommentsServlet.java

```diff
@@ -45,14 +47,20 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     int maxComments = maxString == null ? 5 : Integer.parseInt(maxString);
     int counter = 0;
 
-    List<String> commentHistory = new ArrayList<>();
+    //Multimap<String, String> multimap = ArrayListMultimap.create();
```

Comment:
prefer to delete commented out code before submission

Reply:
Regarding the feature, would that replace the number of comments selection?